app/Http/Controllers/Dashboard/ApiController.php

```diff
@@ -89,4 +110,18 @@ public function getIncidentTemplate()
 
         throw new ModelNotFoundException("Incident template for $templateSlug could not be found.");
     }
+
+    /**
+     * Checks if Cachet is up to date.
+     *
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function checkVersion()
```

Comment:
Or, just `app(Release::class)->...`. That would be just fine too. I think we do that in StyleCI now eveywhere rather than method injection.